examples/config/owner-onboarding-server.yml

```diff
@@ -13,6 +13,10 @@ owner_addresses:
     - dns_name: fdo.example.com
 report_to_rendezvous_endpoint_enabled: false
 bind: 0.0.0.0:8081
+service_info_api_url: "http://localhost:8089/device_info"
+  service_info_api_authentication:
```

Comment:
indentation is off here @7flying - this should be top level

Reply:
I added that using the integration test [example](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/integration-tests/templates/owner-onboarding-server.yml.j2). Which one do you mean that should be top level?